### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by ryan-williams**
Lmk if you want me to just split out the `dmb` fix @danvk

### CODE REVIEW

File: branch/git-delete-merged-branches [hunk 1]
```diff
@@ -2,8 +2,18 @@
 
 # Delete local branches which have been merged into the current one.
 # If you pass --remote, it will prune merged branches instead.
+# An optional final argument gives a remote to check against, defaulting
+# to the DEFAULT_REMOTE env var or "upstream".
 
-remote="${1:-${DEFAULT_REMOTE:-upstream}}"
+# If the last argument exists and doesn't start with "--", interpret it
+# as the name of a remote.
```
Comment (ryan-williams) [inline branch/git-delete-merged-branches]: @danvk lmk if this is better.

also weirdly github is not showing me the usual "commented on an outdated diff" msg from your previous comment, after i killed that commit with a force-push

Comment (danvk) [inline branch/git-delete-merged-branches]: Much clearer, thanks!

A usage line might be more succinct, up to you:

```
# Usage:
# git delete-merged-branches [--remote] [remote-name]
```

**Comment by danvk** [comment]
Did you want to merge this @ryan-williams?

File: branch/git-delete-merged-branches (hunk 1 above)
Comment (ryan-williams) [inline branch/git-delete-merged-branches]: Added a usage line as well!

**Pull request merged**